homeworks/7/InstagramApi/instagramhw/models/ReadRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.example.instagramhw.models;
+
+import jakarta.servlet.http.HttpServletRequest;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class ReadRequest {
+    public static String readRequest(HttpServletRequest request) throws IOException {
+        // Read from request
+        StringBuilder buffer = new StringBuilder();
+        BufferedReader reader = request.getReader();
+        String line;
+        while ((line = reader.readLine()) != null) {
+            buffer.append(line);
+            buffer.append(System.lineSeparator());
+        }
+        return buffer.toString();
+    }
+}

```

Comment:
This class shouldn't belong in the same package as the other models as it isn't a model.